app/services/record_declarations/base.rb

```diff
@@ -18,23 +16,17 @@ def call(params)
       def not_implemented_error
         raise NotImplementedError, "Method must be implemented"
       end
-
-      def schema_validation_params
-        { version: "0.3" }
```

Comment:
How are you going to handle versioning?

Reply:
What do we need versioning for exactly? At the moment?

Reply:
In principle, if (when?) we add `v2` version of the api, we will either need to add a v2 declaration model, or extract v1 validation into their own layer, and add v2 validation in that layer.

We could think of doing something like that now, I just don't know what the value of it is.

Reply:
Just to follow up on it, if we had to add versioning, we would need to have `v1` and `v2` versions of the services that record the declarations. Those services could then have different validation.